<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1" colspan="7">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>图书信息</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>用户信息</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>借还记录</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ShowBtn">
        <property name="text">
         <string>显示</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="9" rowspan="7">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QPushButton" name="submitBtn">
        <property name="text">
         <string>提交修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="revertBtn">
        <property name="text">
         <string>撤销修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deletBtn">
        <property name="text">
         <string>删除选中行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="insertBtn">
        <property name="text">
         <string>插入记录</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="selectBtn">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" rowspan="6" colspan="9">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="7" column="1" colspan="7">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>id：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="idlineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="userlabel">
        <property name="text">
         <string>用户名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userNamelineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="booklabel">
        <property name="text">
         <string>书名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="bookNamelineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="authorlabel">
        <property name="text">
         <string>作者：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="authorNamelineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
